we don't want java touching /etc/mailcap
Alejandro

we can't add new lines to .bin file because it does tail to checksum and to
generate tarball
Alejandro

--- jdk-1_5_0_02-linux-i586-orig.bin	2005-06-04 08:57:18.000000000 +0000
+++ jdk-1_5_0_02-linux-i586.bin	2005-06-04 08:56:36.000000000 +0000
@@ -356,7 +356,7 @@
     if [ -w `dirname ${MAILCAP_FILE}` ]; then
         MC_TEXT="${MC_TEXT}\n${MC_COMMENT}"
         MC_TEXT="${MC_TEXT}\n${MIME_TYPE}; $LATEST_JAVAWS_PATH %s"
-        echo ${echo_args}"${MC_TEXT}" > ${MAILCAP_FILE}
+        #echo ${echo_args}"${MC_TEXT}" > ${MAILCAP_FILE}
     else
         echo "WARNING - cannot write to file:"
         echo "       ${MAILCAP_FILE}"
@@ -380,9 +380,9 @@
     # Create the file if it does not exist
     if [ ! -w ${MIME_FILE} ]; then
-        if [ -w `dirname ${MIME_FILE}` ]; then
-            echo "${NS_COMMENT1}"  > ${MIME_FILE}
-            echo "${NS_COMMENT2}" >> ${MIME_FILE}
-            echo "${NS_COMMENT3}" >> ${MIME_FILE}
-        else
+        if [ ! -w `dirname ${MIME_FILE}` ]; then
+        #    echo "${NS_COMMENT1}"  > ${MIME_FILE}
+        #    echo "${NS_COMMENT2}" >> ${MIME_FILE}
+        #    echo "${NS_COMMENT3}" >> ${MIME_FILE}
+        #else
             echo "WARNING - cannot write to file:"
             echo "       ${MIME_FILE}"
@@ -392,7 +392,7 @@
     fi
     # Add the jnlp entry if it does not already exist.
-    if [ -z "`grep ${MIME_TYPE} ${MIME_FILE}`" ]; then
-        echo ${JNLP_ENTRY} >> ${MIME_FILE}
-    fi
+    #if [ -z "`grep ${MIME_TYPE} ${MIME_FILE}`" ]; then
+    #    echo ${JNLP_ENTRY} >> ${MIME_FILE}
+    #fi
 }
 # End of UpdateMimeTypes
